vendor-onboarding-checklist.md

```diff
@@ -0,0 +1,175 @@
+<li><p>Read GitHub repositories and issues</p></li>
+<li><p>Participate in Mural sprint retrospectives</p></li>
+</ul></td>
+</tr>
+<tr class="even">
+<td>Technical lead</td>
+<td><p>State of Alaska VPN</p>
+<p>SharePoint</p>
+<p>Visual Studio Team Services</p>
+<p>Trello</p>
+<p>Slack</p>
+<p>Alaska DHSS GitHub</p>
+<p>Mural</p>
+<p>Alaska Development Environment (MCI Person Service Test, EIS Program Data Service Test, ARIES Program Data Service Test, Other TBD)</p></td>
+<td><ul>
+<li><p>Access Alaska WAN</p></li>
+<li><p>Access DHSS domain secured resources</p></li>
+<li><p>CRU docs on SharePoint</p></li>
+<li><p>CRU work-items, push branches to VSTS hosted Git repos, create and comment on Git pull-requests, read build output</p></li>
+<li><p>Read and update Trello</p></li>
+<li><p>Read and post messages to Slack</p></li>
+<li><p>Read GitHub repositories and issues</p></li>
+<li><p>Participate in Mural sprint retrospectives</p></li>
+<li><p>Access various non-production services (MCI, EIS, ARIES) to perform development work</p></li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td>Developers</td>
+<td><p>State of Alaska VPN</p>
+<p>SharePoint</p>
+<p>Visual Studio Team Services</p>
+<p>Trello</p>
+<p>Slack</p>
+<p>Alaska DHSS GitHub</p>
+<p>Mural</p>
+<p>Alaska Development Environment (MCI Person Service Test, EIS Program Data Service Test, ARIES Program Data Service Test, Other TBD)</p></td>
+<td><ul>
+<li><p>Access Alaska WAN</p></li>
+<li><p>Access DHSS domain secured resources</p></li>
+<li><p>CRU docs on SharePoint</p></li>
+<li><p>CRU work-items, push branches to VSTS hosted Git repos, create and comment on Git pull-requests, read build output</p></li>
+<li><p>Read and update Trello</p></li>
+<li><p>Read and post messages to Slack</p></li>
+<li><p>Read GitHub repositories and issues</p></li>
+<li><p>Participate in Mural sprint retrospectives</p></li>
+<li><p>Access various non-production services (MCI, EIS, ARIES) to perform development work</p></li>
+</ul></td>
+</tr>
+<tr class="even">
+<td>Researcher</td>
+<td>?</td>
+<td></td>
+</tr>
+<tr class="odd">
+<td>User experience designer</td>
+<td>?</td>
+<td></td>
+</tr>
+<tr class="even">
+<td>Visual designer</td>
+<td>?</td>
+<td></td>
+</tr>
+<tr class="odd">
+<td>Scrum Master</td>
+<td><p>State of Alaska VPN</p>
+<p>Sharepoint</p>
+<p>Visual Studio Team Services</p>
+<p>Trello</p>
+<p>Slack</p>
+<p>Alaska DHSS GitHub</p>
+<p>Mural</p></td>
+<td><ul>
+<li><p>Access Alaska WAN</p></li>
+<li><p>Access DHSS domain secured resources</p></li>
+<li><p>CRU docs on SharePoint</p></li>
+<li><p>Read and update work-items and PR comments in VSTS</p></li>
+<li><p>Read and update Trello</p></li>
+<li><p>Read and post messages to Slack</p></li>
+<li><p>Read GitHub repositories and issues</p></li>
+<li><p>Participate in Mural sprint retrospectives</p></li>
+</ul></td>
+</tr>
+</tbody>
+</table>
+
+# Access procedures
+
+This section outlines procedures to grant
+access.

```

Comment:
We need to add some info here to clarify that access is sponsored and that the DPA PMO sponsors and manages access requests on behalf of vendors.

Reply:
Done.